remote-wallet/src/ledger.rs

```diff
@@ -439,4 +427,66 @@ mod tests {
 
         assert_eq!(pubkey_set.len(), 5); // Ensure keys at various derivation paths are unique
     }
+
+    /// This test can't be run without an actual ledger device connected with the `Ledger Wallet Solana application` running
+    /// It also requires interactive approval of message signing on the ledger.
+    #[test]
+    #[ignore]
+    fn ledger_sign_transaction_test() {
```

Comment:
How about moving this into ledger-app-solana?

Reply:
Removed here. Will make new PR against https://github.com/solana-labs/ledger-app-solana